bootstrap/scss/_custom-styles.scss

```diff
@@ -24,10 +26,247 @@ main {
     background-color: $white;
 }
 
+#main-section {
+    background-image: url(/images/laminas.svg), url(/images/laminas.svg);
+    background-repeat: repeat-y;
+    background-position: 0 0, right 3em top 8em;
+
+    .container {
+        max-width: $max-section-width;
+    }
+}
+
+#homepage {
+    max-width: $max-section-width;
+    margin: 0 auto;
+
+    > * {
+        width: 100%;
+        //width: 95%;
+    }
+}
+
+#homepage-section {
+    margin: 4em auto;
+    display: flex;
+    flex-direction: column;
+    gap: 1em;
+
+    .container {
+        .col {
+            margin-bottom: 4em;
+        }
+    }
+
+    > * {
+        width: 100%;
+    }
+}
+
+.homepage-card {
+    background-color: #FFFFFF;
+    justify-content: space-between;
+    padding: 1.5em;
+}
+
+.homepage-card-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    img {
+        max-width: 250px;
+        min-width: 150px;
+    }
+}
+
+.homepage-card-image {
+    border-radius: 20px;
+    aspect-ratio: 1;
+    width: 100%;
+}
+
+#mezzio-essentials-book {
+    max-height: 290px;
+    transform: rotate(-5deg);
+}
+
+.homepage-card-button {
+    width: fit-content;
+    padding: 5px 15px;
+}
+
+.homepage-card-info {
+    width: 60%;
+}
+
+.homepage-card-image-container {
+    width: 30%;
+    display: flex;
+    justify-content: center;
+}
+
+
+@media(max-width:1150px) {
+    .homepage-card {
+        justify-content: space-evenly;
+    }
+    .homepage-card-image-container {
+        display: none;
+    }
+}
+
+.homepage-title-container {
+    margin: 130px auto;
+}
+
+.page-header, .page-text {
+    color: #{map-get($laminas-colors, laminas)};
+}
+
+.page-header {
+    font-weight: bolder;
+    margin-bottom: 30px;
+}
```

Comment:
This and many more can be configured via Sass variables from Bootstrap. There is no need to introduce so many more classes.

https://getbootstrap.com/docs/5.3/content/typography/#sass-variables